electrum/plugins/hw_wallet/plugin.py

```diff
@@ -276,10 +257,11 @@ def get_password_for_storage_encryption(self) -> str:
     def device_model_name(self) -> Optional[str]:
         """Return the name of the model of this device, which might be displayed in the UI.
         E.g. for Trezor, "Trezor One" or "Trezor T".
+        If this method is not defined for a plugin, the plugin name is used as default
         """
-        return None
+        return self.plugin.name
 
-    def manipulate_keystore_dict_during_wizard_setup(self, d: dict) -> None:
+    def manipulate_keystore_dict_during_wizard_setup(self, d: dict) -> None:  # noqa: B027
```

Comment:
It is suspicious that this method is only called from base_wizard.py, and not from the new wizard.

Reply:
This is only used for coldcard, which isn't fully ported yet.

Reply:
Thx, committed